<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMNGMainWindow</class>
 <widget class="QMainWindow" name="PMNGMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMNGMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="qvlay_leftPane"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="qvlay_compositionEFG">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="ql_signal">
            <property name="text">
             <string>Profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="qcbox_signal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ql_plotCutoff">
            <property name="text">
             <string>Plot cutoff (min)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FloatingValueLineEdit" name="qle_plotCutoff">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcb_autoPlotCutoff">
            <property name="text">
             <string>Auto</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ql_injZoneLength">
            <property name="text">
             <string>Inj. zone length (mm)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FloatingValueLineEdit" name="qle_injZoneLength">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qpb_replotEFG">
            <property name="text">
             <string>Replot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>E&amp;xport</string>
    </property>
    <widget class="QMenu" name="menuElectrophoregram">
     <property name="title">
      <string>&amp;Electrophoregram</string>
     </property>
     <addaction name="actionExportEFGAsCSV"/>
    </widget>
    <addaction name="menuElectrophoregram"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionDatabase_editor"/>
    <addaction name="actionOpen_database"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Ad&amp;vanced</string>
    </property>
    <addaction name="actionSet_debugging_output"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExportEFGAsCSV">
   <property name="text">
    <string>&amp;As CSV</string>
   </property>
  </action>
  <action name="actionDatabase_editor">
   <property name="text">
    <string>&amp;Database editor</string>
   </property>
  </action>
  <action name="actionOpen_database">
   <property name="text">
    <string>&amp;Open database</string>
   </property>
  </action>
  <action name="actionSet_debugging_output">
   <property name="text">
    <string>&amp;Set debugging output</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/ui/elementaries/floatingvaluelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
